kube/src/api/resource.rs

```diff
@@ -34,46 +34,46 @@ pub struct Resource {
 
 impl Resource {
     /// Cluster level resources, or resources viewed across all namespaces
-    pub fn all_with<K: Meta>(f: &K::Family) -> Self {
+    pub fn all_with<K: Meta>(rtt: &K::DynamicType) -> Self {
         Self {
-            api_version: K::api_version(f).into_owned(),
-            kind: K::kind(f).into_owned(),
-            group: K::group(f).into_owned(),
-            version: K::version(f).into_owned(),
+            api_version: K::api_version(rtt).into_owned(),
+            kind: K::kind(rtt).into_owned(),
+            group: K::group(rtt).into_owned(),
+            version: K::version(rtt).into_owned(),
             namespace: None,
         }
     }
 
     /// Namespaced resource within a given namespace
```

Comment:
```suggestion
    /// Namespaced resource within a given namespace
    ///
    /// This function accepts `K::DynamicType` so it can be used with dynamic resources.
```